### PR TITLE
Add new information to Analytics guide

### PR DESCRIPTION
**Pull request by szekelyzol**
Changes are for these Asana tasks:
* [Task about new metrics](https://app.asana.com/0/1205634133195403/1208145821422434)
* [Task about explaining the difference between timeseries and aggregations](https://app.asana.com/0/1205634133195403/1208119928333952)

**Summary**

- Reorganized the Analytics guide to focus more on usage and examples
- Added new use cases based on the new metrics
- Added explanation about user sessions
- Added detailed explanation about the difference between the `/data/metrics/{metric}/{aggregation}` and `/data/timeseries/{metric}` endpoints

### CODE REVIEW

**Comment by szekelyzol** [comment]
Hi folks!

* @mioara-api-video if you have time, please help with a general review of the new content 📃 
* @ThibaultBee please help me confirm that the information about **sessions** is correct 🙏 
* @GuillaumeDGBN please check that the explanation of the difference between the `/data/metrics/{metric}/{aggregation}` and `/data/timeseries/{metric}` endpoints is acceptable or not 🙂 

Thanks everyone!

**Comment by szekelyzol** [comment]
@ThibaultBee I updated the information about sessions. Thanks!

File: analytics/overview.md [hunk 1]
```diff
@@ -67,17 +41,111 @@ Here are some real-world questions where metrics and dimensions from Analytics c
...
+
+These 2 endpoints return data that is not limited to 30 days of retention, and can return data from earlier versions of Analytics.
+</Callout>
+
+### Sessions
+
+User sessions affect how data like the number of viewers is counted. 
+
+* For web users, a browser tab counts as one user session. 
+* For mobile users, a session starts when an app is launched. Apps going in the background remain in session. A new session starts when an app stops or is destroyed, and then launches again.
```
Comment (ThibaultBee) [inline analytics/overview.md:85]: `app stops or is destroyed` to be replaced with `app is destroyed`.

File: analytics/overview.md [hunk 2]
```diff
@@ -67,17 +41,111 @@ Here are some real-world questions where metrics and dimensions from Analytics c
...
 
+## How it works
+
+Analytics uses player events to analyze and segment your viewers' interactions with your content. Here are some key aspects:
+
+- By default, api.video retains analytics data for 30 days. You can extend data retention to 3 months or 12 months through the [Analytics page in the Dashboard](https://dashboard.api.video/analytics) - click on the `3M` or `12M` buttons to get started!
+- Player events are generated when your viewers engage with a video or live stream session.
+- Data is refreshed in real time, with a frequency of `<5s`.
+- Data does not carry over from the previous versions of Analytics, and unique viewer data starts only from `2024-08-14`.
+- `unique` viewers are only unique for a day.
```
Comment (GuillaumeDGBN) [inline analytics/overview.md:67]: not really, unique viewer is calculated each time regarding the time period selected. 
If you select 1 month period, you will have unique viewer on 1 month.

Comment (szekelyzol) [reply analytics/overview.md:67]: As confirmed by Sébastien, `unique` is only counted for 24 hours.

File: analytics/overview.md (hunk 1 above)
Comment (szekelyzol) [reply analytics/overview.md:85]: Fixed ✅

**Review by GuillaumeDGBN — APPROVED** [review]

**Pull request merged**